x-pack/plugins/stack_connectors/server/connector_types/email/send_email.ts

```diff
@@ -92,6 +92,13 @@ export async function sendEmailWithExchange(
   const { transport, configurationUtilities, connectorId } = options;
   const { clientId, clientSecret, tenantId, oauthTokenUrl } = transport;
 
+  let tokenUrl = oauthTokenUrl;
+  if (!tokenUrl) {
+    const exchangeUrl =
+      configurationUtilities.getMicrosoftExchangeUrl() ?? EXCHANGE_ONLINE_SERVER_HOST;
```

Comment:
could using `EXCHANGE_ONLINE_SERVER_HOST` and `GRAPH_API_OAUTH_SCOPE` in config as default value be better or more clear.

Reply:
That makes sense! 👍

Reply:
Updated in https://github.com/elastic/kibana/pull/175812/commits/44dd46f8ddf0ef25ed2ca50b45b450a6f229356f